### PR TITLE
Signup page created with simple validation

### PR DESCRIPTION
**Pull request by nicholasrobinharrison**
Not sure how to change text field outline colour, and button hover colour in material-ui. Placeholder submit function until API is ready to go.

### CODE REVIEW

**Review by rkerins91 — APPROVED** [review]
Looks great, we should probably have a name or username field on the signup form.

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        }}
+      />
+      <br />
+      <Button
+        className={classes.button}
+        variant="contained"
+        color="primary"
+        onClick={submit}
+      >
+        Sign Up
```
Comment (ashan36) [inline client/src/pages/Login.js]: Looks great. Button for logging in should say "Login" instead of "Sign Up"

**Review by ashan36 — CHANGES_REQUESTED** [review]
Just update that button text and the rest looks great.

Also is it possible to make the background a gradient to look more like the mockup?

```
background: linear-gradient(to top, rgba(87,40,184,1) 15%, rgba(119,140,255,1) 100%);
```
Or something like that.

**Comment by nicholasrobinharrison** [comment]
> Just update that button text and the rest looks great.
> 
> Also is it possible to make the background a gradient to look more like the mockup?
> 
> ```
> background: linear-gradient(to top, rgba(87,40,184,1) 15%, rgba(119,140,255,1) 100%);
> ```
> 
> Or something like that.

I set the gradient as be the same colours as in the style guide! 
Purple color: #6E3ADB
Dark Purple: #501CBD
It's possible that they are just too close together to tell apart, 
I've updated the gradient with your suggestion and it does look more like the markup

File: client/src/pages/Login.js (hunk 1 above)
Comment (nicholasrobinharrison) [reply client/src/pages/Login.js]: Changed

**Review by ashan36 — APPROVED** [review]
Nice!

File: client/src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  const [password, setPassword] = useState("");
+
+  const submit = () => {
+    return; // placeholder
+  };
+
+  return (
+    <SignUpContainer>
+      <Typography className={classes.text}> Sign In </Typography>
+      <br />
```
Comment (skassam21) [inline client/src/pages/Login.js]: Let's avoid using breaklines, it's generally bad practice. Instead we use padding or margin. In Typography there is paragraph + gutterBottom

File: client/src/pages/Signup.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      />
+      <br />
+      <TextField
+        className={classes.input}
+        error={validatePassword()}
+        label="password"
+        type="password"
+        variant="outlined"
+        helperText={
+          password.length > 0 && password.length < 6
```
Comment (skassam21) [inline client/src/pages/Signup.js]: This seems a bit repetitive with the validatepassword function... and also 6 is kind of a magic number, maybe you want to consolidate that information in this file - you could make validatePassword return text or null, and then have error as true if it returns a value

File: client/src/pages/Login.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        }}
+      />
+      <br />
+      <TextField
+        className={classes.input}
+        label="password"
+        type="password"
+        variant="outlined"
+        helperText={
+          password.length > 0 && password.length < 6 ? "error text" : ""
```
Comment (skassam21) [inline client/src/pages/Login.js]: We don't really need this validation in the login page

File: client/src/pages/Signup.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+      />
+      <br />
+      <TextField
+        className={classes.input}
+        error={validateSecondPassword()}
+        label="re-enter password"
+        type="password"
+        variant="outlined"
+        helperText={
+          secondPassword.length > 0 && password !== secondPassword
```
Comment (skassam21) [inline client/src/pages/Signup.js]: same as above

**Review by skassam21 — COMMENTED** [review]
Looks good in general!!!

File: client/src/pages/Signup.js (hunk 3 above)
Comment (nicholasrobinharrison) [reply client/src/pages/Signup.js]: Great, I'll update this

**Pull request merged**